src/platform-integration/web/faq.md

```diff
@@ -110,6 +110,55 @@ with the location of your hosted HTML page:
 <iframe src="URL"></iframe>
 ```
 
+Another way of embedding is In your HTML file where you want to embed the Flutter app,
+create a container using a <div> element. Give it an id so that you can reference it in
+JavaScript. 
+
+```html
+<div id="flutter-container"></div>
+```
+In the same HTML file, include the JavaScript files generated by Flutter in the <body> section. 
+Add the following scripts:
+```html
+<script src="path/to/flutter.js"></script>
+<script src="path/to/flutter_assets/main.dart.js"></script>
+```
+Replace "path/to/flutter.js" and "path/to/flutter_assets/main.dart.js" with the actual paths to your
+Flutter-generated JavaScript files.
+
+Add JavaScript code to initialize your Flutter app inside a <script> tag below the Flutter script includes. Use the flutter.runApp 
+function to specify the entry point of your Flutter app.
+
+```html
+<script>
+  flutter.runApp(
+    {
+      element: document.getElementById('flutter-container'),
+      initialRoute: '/',
+      routes: {
+        '/': (_) => MyApp(),
+      },
+    },
+  );
+</script>
+```
+
+In the code above, replace '/' with the appropriate initial route of your Flutter app, and replace MyApp()
+with the main widget of your Flutter app.
```

Comment:
**chore:** 
* Adjust for list and code block spacing.
* Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading). 
* Write as condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).

```suggestion
1. To initialize your Flutter app, add the following JavaScript code.
   Place this code in a `<script>` tag after the `<script>` tags from the previous step.
   To specify the entry point of your Flutter app, use the `flutter.runApp` 
function.
    Use the `id` you set in the second step of this procedure.

   ```html
    <script>
     flutter.runApp(
       {
         element: document.getElementById('flutter-container'),
         initialRoute: '/',
         routes: {
           '/': (_) => MyApp(),
         },
       },
     );
   </script>
   ```

   Substitute two values in the preceding code block:
   
   * Substitute `/` with the appropriate initial route of your Flutter app
   * Substitute `MyApp()` with the main widget of your Flutter app.
```